src/VisualStudio/Core/Def/Implementation/Options/RoamingVisualStudioProfileOptionPersister.cs

```diff
@@ -190,6 +177,24 @@ public bool TryFetch(OptionKey optionKey, out object value)
             return true;
         }
 
+        private bool DeserializeCodeStyleOption<T>(ref object value)
+        {
+            if (value is string serializedValue)
+            {
+                try
+                {
+                    value = CodeStyleOption<T>.FromXElement(XElement.Parse(serializedValue));
+                    return true;
+                }
+                catch (Exception)
```

Comment:
So for the older roaming options that were true/false and are now a tri-state, should we have some affordance here to convert the older values to the newer values?